docs/README.md

```diff
@@ -97,6 +97,6 @@ $ atk dev -ua -s dev --optionals gpus vnc
 WARNING  | logger.set_verbosity :: Verbosity has been set to WARNING
 [+] Running 1/1
  ✔ Container art-dev  Started
-art@art-dev:~/art/workspace$ colcon build --symlink-install
+art@art-dev:~/art/workspace$ colcon build --symlink-install --packages-up-to art_dev_meta
```

Comment:
may be nice to add a desc/link for symlinks and a link to the meta packages page from here. If you're tired of writing docs tho I can do it in a different MR :)

Reply:
i say at the beginning of the quick start section to look through the TOC first and read through all the docs. I think this section is more of a quick reference for us when we forget how to run a command (but we already know what it's doing). I added the link below though.